apps/nextjs-website/react-components/components/Accordion/Accordion.tsx

```diff
@@ -36,42 +30,34 @@ const Accordion = (props: AccordionProps) => {
       {...(sectionID && { id: sectionID })}
     >
       <Container>
-        <Grid item xs={12}>
-          <Stack px={{ xs: 1, md: 0 }} spacing={{ xs: 2, md: 5 }}>
-            {/** Title */}
-
-            <Title
-              variant='h4'
-              textColor={textColor}
-              title={title}
-              textAlign={textAlignment}
-            />
-
-            {/** Subtitle */}
-
-            {subtitle && (
-              <Subtitle
-                variant='h6'
-                textColor={textColor}
-                subtitle={subtitle}
-                textAlign={textAlignment}
-              />
-            )}
-
-            {/** Description */}
-            {description && (
-              <Body
-                variant='body2'
+        <Grid container spacing={4} alignItems='flex-start'>
+          <Grid item xs={12} md={layout === 'center' ? 12 : 4}>
+            <Stack px={{ xs: 1, md: 0 }} spacing={{ xs: 2, md: 4 }}>
+              <Title
+                variant='h4'
                 textColor={textColor}
-                body={description}
+                title={title}
                 textAlign={textAlignment}
```

Comment:
textAlignment should only be taken into consideration when layout = 'center'

Otherwise, default to 'left'